src/imagelightbox.js

```diff
@@ -478,10 +432,26 @@
                 }
                 inProgress = false;
                 _onStart();
+		build();
```

Comment:
One thing that needs to be done, is testing creation of two sets with *different* options. Gonna test that out.